src/query/expression/src/values.rs

```diff
@@ -1506,62 +1506,9 @@ impl ColumnBuilder {
     }
 
     pub fn repeat(scalar: &ScalarRef, n: usize, data_type: &DataType) -> ColumnBuilder {
```

Comment:
Using `push_repeat` to implement the `repeat` function does not seem to be a good idea, because it will reduce performance, it is recommended to keep the original implementation. :)

Reply:
I've checked the implementations of `push_repeat` and `repeat` in most of builder and there's almost no difference, maybe I'm missing something?

Reply:
For example, for nullable columns, we can construct a `Bitmap` directly, but `push_repeat` will call `push_null`.

Reply:
You can check out the official Rust documentation: https://doc.rust-lang.org/std/vec/struct.Vec.html
```rs
let vec = vec![0; 5];
assert_eq!(vec, [0, 0, 0, 0, 0]);

// The following is equivalent, but potentially slower:
let mut vec = Vec::with_capacity(5);
vec.resize(5, 0);
assert_eq!(vec, [0, 0, 0, 0, 0]);
```

Reply:
Yes, 10% slower roughly
```rust
#[cfg(test)]
mod tests {
    extern crate test;
    use test::Bencher;

    use super::*;

    #[bench]
    fn bench_repeat(b: &mut Bencher) {
        let scalar = Scalar::String(String::from("hello"));
        let scalar = &scalar.as_ref();

        b.iter(|| {
            let builder = ColumnBuilder::repeat(scalar, 100, &DataType::String);
            builder.build()
        })
    }

    #[bench]
    fn bench_repeat2(b: &mut Bencher) {
        let scalar = Scalar::String(String::from("hello"));
        let scalar = &scalar.as_ref();

        b.iter(|| {
            let n = 100;
            let mut builder = ColumnBuilder::with_capacity_hint(&DataType::String, n, false);
            builder.push_repeat(scalar, n);
            builder.build()
        });
    }
}
```